nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/checkpoint/CheckpointConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */package org.apache.nifi.processors.azure.eventhub.checkpoint;
+
+public final class CheckpointConstants {
```

Comment:
Rather than having a generalized `Constants` class, it is better to use one or more enums that provide a clear purpose. In this case, I recommend a `CheckpointStateKey` enum for `CLIENT_ID` and `CLUSTERED`, and a `StoreKeyPrefix` enum for `OWNERSHIP` and `CHECKPOINT`. In the case of `StoreKeyPrefix`, the enum could have a method named `getPrefix()` that would return the lowercase value.

Reply:
I introduced enums but used a bit different names for them: `CheckpointStoreKey` and `CheckpointStoreKeyPrefix` because both are related to keys in the checkpoint store.
I also added a field for the key literal in `CheckpointStoreKey` too because I preferred to use specific keys in the store instead of the raw enum names.
I hope it will be fine. Let me know if further adjustments are needed.

Reply:
Thanks @turcsanyip, the names you selected look good. Will review once more pending automated build completion, but this looks good.